### PR TITLE
LOC-26 Strings: integrate English edits from docs team

### PR DESCRIPTION
**Pull request by pdeuter**
@anamartinez @ydrain @maximeprades @chnorton @jwswj 

These analogous changes need to be made elsewhere in the magento_extension project, especially the key change for "text.integrations.magento.config.test_connection.title".  Please make these changes soon, so that our next sweep will be clean.  thx in advance

### CODE REVIEW

File: src/app/locale/en_US/Zendesk_Zendesk.yml [hunk 1]
```diff
@@ -179,7 +179,7 @@ parts:
         title: "Configuration form: button to save the form"
         value: "Save Config"
   - translation:
-        key: "text.integrations.magento.config.test_connection"
+        key: "text.integrations.magento.config.test_connection.title"
```
Comment (anamartinez) [inline src/app/locale/en_US/Zendesk_Zendesk.yml:182]: Do we need to change the key in the code as well?

Comment (pdeuter) [inline src/app/locale/en_US/Zendesk_Zendesk.yml:182]: Yes, the key needs to be changed in the code too.

Comment (jwswj) [inline src/app/locale/en_US/Zendesk_Zendesk.yml:182]: Code doesn't actually use our keys, it uses the English version as "the key".

**Comment by jwswj** [comment]
:+1:

**Pull request merged**